ee-9/pom.xml

```diff
@@ -673,6 +674,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.spec.jakarta.xml.soap</groupId>
+                <artifactId>jboss-saaj-api_2.0_spec</artifactId>
+                <version>${version.org.jboss.spec.jakarta.xml.soap.saaj-api_2.0_spec}</version>
```

Comment:
I miss the license entry for this new artifact. The ee-8-api version has one for `org.jboss.spec.javax.xml.soap:jboss-saaj-api_1.4_spec`. I would expect the same for the new one.

Reply:
Added this entry. Thanks for review @yersan